tests/parser_tests.py

```diff
@@ -527,6 +530,25 @@ def test_parse_with_extra_words_at_start_and_end_valid(self):
             datetime(2016, 5, 16, 4, 5, 6, 789120),
         )
 
+        self.assertEqual(
+            self.parser.parse(
+                "Meet me at my house on the my birthday (2019-24-11)", "YYYY-DD-MM"
+            ),
+            datetime(2019, 11, 24),
+        )
+
+        self.assertEqual(
+            self.parser.parse(
+                "Monday, 9. September 2019, 16:15-20:00", "dddd, D. MMMM YYYY"
```

Comment:
While this is the exact English translation of the buggy case described in https://github.com/crsmithdev/arrow/issues/701, it is always good to include the original, unaltered problematic test case in the regression tests to make sure that we nailed down the original issue. You can simply add it below this one.

Reply:
I put it in factory_tests.py because specifying locale can't be done within parser.py